tests/numerics/ad/test_operator_functions.py

```diff
@@ -0,0 +1,78 @@
+"""Test collection testing Ad operator functions.
+
+Testing involves their creation, arithmetic overloads and parsing.
+
+"""
+from __future__ import annotations
+
+import pytest
+
+import numpy as np
+import porepy as pp
+
+
+def test_ad_function():
+    """Tests involving the Ad wrapper for analytically represented functions.
+    
+    Note:
+        Value and Jacobian of various analytically represented functions are covered
+        in test_functions.py.
+        
+        This test tests the AD wrapper ad.Function and its wrapping functionality using
+        the identity function.
+    
+    """
+
+    func = lambda x: x  # identity
+
+    F = pp.ad.Function(func, 'identity')
+
+    g = pp.CartGrid(np.array([3, 2]))
+    mdg = pp.meshing.subdomains_to_mdg([g])
+    mdg.compute_geometry()
+
+    sds = mdg.subdomains()
+    eqs = pp.ad.EquationSystem(mdg)
```

Comment:
Can we reserve "eqs" for "equations" and use e.g. "eqsys" for an EquationSystem?